core/src/core-api/auth.js

```diff
@@ -66,6 +66,7 @@ class LuigiAuth {
    * @property {string} idToken - id token, used for renewing authentication
    */
   get store() {
+    if(!LuigiConfig.initialized)  console.warn("Luigi Core is not initialized yet. Consider moving your code to the luigiAfterInit lifecycle hook. Documentation: https://docs.luigi-project.io/docs/lifecycle-hooks");
```

Comment:
We prefer not to have those shorthands :-) and max line length should be taken into account. 
Something similar to:
```
    if(!LuigiConfig.initialized) {
      console.warn(`Luigi Core is not initialized yet. Consider moving your code to the luigiAfterInit lifecycle hook. 
      Documentation: https://docs.luigi-project.io/docs/lifecycle-hooks`);
    }
```